### ISSUE TITLE
Mise en page commune pour les différents documents du langage

### ISSUE THREAD

**Issue by flodavid** [ShihoWasTaken/ezlanguage#28]
Il faudrait se mettre d'accord sur un style visuel commun pour les documents décrivant le langage, peut être avec Organisation. Style que l'on pourrait éventuellement réutilisé lorsque l'on crée des documents au sein de nos groupes respectifs.

**Comment by mtroysi** [comment]
Je suis d'accord. Je pense qu'il faudrait également intégrer le groupe Compilation sur ce problème : ce sont eux les principaux destinataires de nos documents, il serait donc judicieux d'avoir leur avis sur les documents reçus jusqu'ici, leurs préférences, etc.
